ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -850,7 +850,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = eu.jhass.insporation;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AppStore eu.jhass.insporation";
+				PROVISIONING_PROFILE_SPECIFIER = "match Development eu.jhass.insporation";
```

Comment:
Not sure we should commit this every time we change it locally for testing?

Reply:
This helps (me) running locally on device. For Appstore this is changed by fastline script. 
I am still trying to push this provisioning profile to the private repo. ..

Reply:
This doesn't affect testing on simulator.

Reply:
I just noticed you've been changing these values around basically every other commit as a drive-by change :P

Reply:
I am always trying hard not to checkin this - ..

Reply:
That's why I always `git commit -p`, makes it easy to skip things like this and also forces me to review what I've done once more :)

Reply:
'Use the interactive patch selection interface' - I'll try this next time.